care/utils/tests/test_base.py

```diff
@@ -7,6 +7,7 @@
 from django.contrib.gis.geos import Point
 from django.utils.timezone import make_aware
 from pytz import unicode
+from rest_framework import status
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5324897)